cost/aws/reserved_instances/compute_purchase_recommendation/README.md

```diff
@@ -0,0 +1,69 @@
+## AWS Reserved Instances Recommendations & Purchase Policy Template
+
+## NOTE: These RI Purchase Recommendations are generated by AWS. This policy will raise incidents based on those recommendations.
+
+### What it does
+This Policy Template leverages the [AWS Cost Explorer API](https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_GetReservationPurchaseRecommendation.html). It will raise incidents if AWS has any RI Purchase Recommendations, whose net savings exceeds the *Net Savings Threshold* parameter in the Policy.
```

Comment:
Include that there is the option to purchase the RI after approval.  Use the same text in the policy short_description without the links to the readme or docs.